tests/boot-test.js

```diff
@@ -1,4 +1,4 @@
-require( 'babel-core/register' );
+require( '@babel/register' );
```

Comment:
I removed this line altogether locally and things didn't break :man_shrugging: 

I guess it's because this file doesn't include any syntax that current node versions don't understand anyway (e.g. `const` is fine).

Reply:
`@babel/register` intercepts the `require` call and transpiles the loaded file before passing it to Node.js. Jest has a custom fancy module loader that does something similar.

Today, when Node.js has full up-to-date ES2018 support, this is needed only for `import` and `export` statements, JSX and other custom syntax.